src/modules/drawing/components/DrawingToolbox.vue

```diff
@@ -67,6 +75,20 @@ function onCloseClearConfirmation(confirmed) {
         store.dispatch('setDrawingMode', { mode: null, ...dispatcher })
     }
 }
+
+async function onSaveName() {
+    if (activeKmlLayerName.value === activeKmlLayer.value.name) return
+    store.dispatch('updateLayer', {
+        index: activeKmlLayerIndex.value,
+        layer: {
+            id: activeKmlLayer.value.id,
+            name: activeKmlLayerName.value,
```

Comment:
Wouldn't it make more sense to sanitize it here, instead of only when exporting it?